libs/core/src/lib/checkbox/checkbox/checkbox.component.html

```diff
@@ -6,6 +6,13 @@
     [name]="name"
     [disabled]="disabled"
     [indeterminate]="isIndeterminate"
+    [attr.tabIndex]="tabIndex"
+    [attr.aria-label]="ariaLabel || null"
+    [attr.aria-labelledby]="ariaLabelledby"
+    [attr.aria-checked]="_getAriaChecked()"
```

Comment:
Instead of using the function call `_getAriaChecked()` here, you should create a component property here, `ariaChecked` and then update it's value in the `nextValue()` method.

```
public ariaChecked: 'true' | 'false' | 'mixed';
```

Reply:
Done